src/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -114,12 +109,6 @@ public void TestGetType(Type type)
         }
 
         [Fact]
-        public void TestToString()
-        {
-            Assert.Equal("System.Runtime.Tests.dll", Module.ToString());
```

Comment:
Right, but my question was why you aren't simply disabling the tests for AOT, rather than deleting them. This API certainly works for CoreCLR, hence the test.